include/jemalloc/internal/hpa_opts.h

```diff
@@ -58,7 +58,7 @@ struct hpa_shard_opts_s {
 
 #define HPA_SHARD_OPTS_DEFAULT {					\
 	/* slab_max_alloc */						\
-	64 * 1024,							\
+	64 * 1024,						\
```

Comment:
Decided not to change the default for now. This change is a format problem, will drop